src/components/StickySidebar.vue

```diff
@@ -0,0 +1,26 @@
+<template>
+  <div
+    ref="sidebar"
+    style="height: 1px;"
+    class="col-12 col-lg-3 float-left hide-sm hide-md hide-lg"
```

Comment:
This should not be part of the component, we may want to use that component on right sidebar too or on the main content.

Reply:
Yes, I wanted to do that at first but then there would be extra javascript needed in the parent component. 

Actually though, to put the sidebar on the right there are no changes needed here, I made another commit that shows an example.

Reply:
Can't you access parent component from Sticky component?

Reply:
I tried it, and I get `clientWidth` undefined, the `ref` seems to be scoped.

Reply:
I also tried `$parent.$refs` same error.

Reply:
But as I said moving the sidebar to the right doesn't require any changes in this bit of code, any other cases we could handle with props.

Reply:
The right sidebar don't have same size, it doesn't use `col-lg-3` it use `col-lg-4` (on Create.vue or everywhere else). Also the component doesn't cover the case where we want sticky menu on the main content.

Reply:
Is there an example I can look at where the main content is sticky, because I'm not sure I'm getting the right picture.

Reply:
For example i would like this to be sticky too
![image](https://user-images.githubusercontent.com/16245250/105227907-74ee6500-5b94-11eb-8a0b-82294261db7c.png)
https://snapshot.page/#/yam.eth

Reply:
I think that will need to be a separate component as the sidebar never moves, but this will have to move initially and then become fixed at the bottom of the navbar.

Reply:
I might be wrong, still figuring things out along the way.

Reply:
c2ad6d5 is the cleanest solution I could come up with so far.

Reply:
Sorry I mean 7ce10fd